syntax/markdown-latex-combined-license.txt

```diff
@@ -0,0 +1 @@
+<html><body>You are being <a href="https://raw.githubusercontent.com/microsoft/vscode-markdown-tm-grammar/main/LICENSE.txt">redirected</a>.</body></html>
```

Comment:
It would be better to copy the file itself instead of writing a link to it.

Reply:
My bad! I messed up the `curl` command when downloading the license file.